backend/app/users/models.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from sqlalchemy import TIMESTAMP, Integer, String, text
+from sqlalchemy.orm import Mapped, mapped_column
+
+from ..models import Base
+
+
+class Users(Base):
+    """ORM for managing user information."""
+
+    __tablename__ = "users"
+
+    user_id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
```

Comment:
Maybe for a future PR, we should keep track of when the user logged in and logged out. Probably in a separate table.